app/services/versioned_files_service/delete_action.rb

```diff
@@ -1,35 +1,15 @@
 class VersionedFilesService
```

Comment:
This doesn't seem to have anything to do with versions anymore. Can it be renamed?